pkg/kubelet/config/file.go

```diff
@@ -200,3 +201,21 @@ func (s *sourceFile) replaceStore(pods ...*v1.Pod) (err error) {
 	}
 	return s.store.Replace(objs, "")
 }
+
+func checkPlainTextFile(file string) (bool, error) {
+	out, err := exec.Command("file", "--mime-type", file).CombinedOutput()
+	if err != nil {
+		return false, fmt.Errorf("failed to exec file command: %v", err)
+	}
+
+	lines := strings.Split(string(out), ": ")
```

Comment:
If the manifest filename contains ": ", then the output string will have at least two ": ".

for example,

```shell
$ touch "file1: file1"
$ echo "test" > "file1: file1"
$ file --mime-type "file1: file1"
file1: file1: text/plain
```
 The length of the split result will be 3, not 2. However this should be considered valid input.

To avoid this circumstance, we just use brief mode not to prepend filenames to the output lines.

```shell
$ file -b --mime-type "file1: file1"
text/plain
```